source/configuration_manual/plugins/nfs_hostchange_plugin.rst

```diff
@@ -0,0 +1,44 @@
+=====================
+NFS Hostchange plugin
+=====================
+
+.. note::
+
+   This is a Dovecot Pro only plugin.
+
+
+This plugin is intended for NFS systems. It optimizes host changing by keeping
+user indexes on local disk instead of NFS.
```

Comment:
The README file contains a lot more explanations, which I think are useful. Should include them in doc.dovecot.org also.

Reply:
Used the README file instead.